app/enterprise/1.3-x/deployment/installation/centos.md

```diff
@@ -1,255 +1,328 @@
 
-4. Exit from PostgreSQL
+7. Restart PostgreSQL.
 
-    ```
-    $ \q
+    ```bash
+    $ sudo systemctl restart postgresql-9.6
     ```
 
-5. Return to terminal
+## Step 4. Modify Kong's configuration file to work with PostgreSQL
 
-    ```
-    $ exit
+1. Make a copy of Kong's default configuration file.
+
+    ```bash
+    $ sudo cp /etc/kong/kong.conf.default /etc/kong/kong.conf
     ```
 
-6. Run the following command to access the PostgreSQL configuration file.
+2. Uncomment and update the PostgreSQL database properties in `/etc/kong/kong.conf` using your preferred text editor. Replace pg_user, pg_password and pg_database with the values: 
 
     ```
-    $ sudo vi /var/lib/pgsql/9.5/data/pg_hba.conf
+    pg_user = kong
+    pg_password = kong
+    pg_database = kong
     ```
 
-7. Under IPv4 local connections replace `ident` with `md5`
+## Step 5. Seed the Super Admin password and boostrap Kong
 
-    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
-    | # IPv4 local connections:                      |
-    | host | all      | all  | 127.0.0.1/32 | **md5**|
-    | # IPv6 local connections:                      |
-    | host | all      | all  | ::1/128      | ident  |
+Setting a password for the **Super Admin** before initial start-up is strongly recommended. This will permit the use of RBAC(Role Based Access Control) at a later time, if needed.
 
-8. Restart PostgreSQL
+1. Create an environment variable with the desired **Super Admin** password and keep password in a safe place:
 
-    ```
-    $ sudo systemctl restart postgresql-9.5
+    ```bash
+    $ export KONG_PASSWORD=<password-only-you-know>
     ```
 
+2. Run migrations to prepare the Kong database.
 
-## Step 5. Modify Kong's configuration file
-
-To use the newly provisioned PostgreSQL database, Kong's configuration file
-must be modified to accept the correct PostgreSQL user and password.
+    ```bash
+    $ sudo /usr/local/kong migrations bootstrap -c /etc/kong/kong.conf
+    ```
 
-1. Make a copy of the default configuration file
+3. Start Kong Enterprise:
 
+    ```bash
+    $ sudo /usr/local/bin/kong start -c /etc/kong/kong.conf
     ```
-    $ cp /etc/kong/kong.conf.default /etc/kong/kong.conf
+
+4. Verify Kong Enterprise is working:
+
+    ```bash
+    $ curl -i -X GET --url http://localhost:8001/services
     ```
+    
+5. You should receive a `HTTP/1.1 200 OK` message.
+    
+## Step 6. Finalize your Configuration and Verify Kong was Successfully Installed
+
+### Enable and Configure Kong Manager
 
-2. Uncomment and update the PostgreSQL database properties inside the Kong conf:
+1. To access Kong Enterprise's Graphical User Interface, Kong Manager, update the `admin_gui_url` property in `/etc/kong/kong.conf` file the to the DNS, or IP address, of the CentOS system. For example:
```

Comment:
something missing in this part of the sentence "file the to the DNS, or IP address"